testing-guidance/workflow-tests/ragdoll/ragdoll-authoring-improvements.md

```diff
@@ -0,0 +1,47 @@
+| **Self-collision Warning** | <ol><li>Make sure that the Ragdoll Collider drawing is enabled under the "eye" button dropdown menu.</li><li>See if there are any bones present with red wiremeshes. On the skeleton set up in these workflows, these should be "L_leg_INT", and "R_leg_INT". Red coloration of the wiremesh means that two joints **which are not adjecent in the hierarchy** (for example "R_knee_INT" bone is adjacent to "R_leg_INT", and "R_foot_INT") are colliding with each other. This can be fixed by removing the collision of one of the intersecting bones (select a bone in the Skeleton Outliner, and choose Ragdoll → Remove colliders), or by simply moving/rotating the bones so that they do not intersect, by changing their Offset/Rotation properties. </li> <li>Select a bone, and move/rotate it around by modifying the collider's Offset/Rotation properties under the Ragdoll pane, or using relevant manipulators in the Atom Render Window. Note the change of the wiremeshes color from grey to red when intersecting with other bones that are not adjacent in the bone hierarchy.</li></ol> | <ul><li>Ragdoll Collider drawing can be enabled, and draws wiremeshes of the bones added to the ragdoll.</li><li>Intersecting non-adjacent in hierarchy bones have red wiremeshes instead of grey.</li><li>Moving/rotating bones around properly updates the wiremesh coloration if an unallowed intersection occurs.</li></ul>  |
```

Comment:
Step 2: Generic this to a user's actor and reference Rin as an example.

Step 3: Move the "Note"/Validation to Things to Watch Out For.

Reply:
Adjusted steps 2, and 3 as requested.